core/variable_map.js

```diff
@@ -358,9 +358,6 @@ VariableMap.prototype.getVariablesOfType = function(type) {
 VariableMap.prototype.getVariableTypes = function(ws) {
   const variableMap = {};
   object.mixin(variableMap, this.variableMap_);
```

Comment:
While you're modifying this function, please change it to [use `Object.assign` instead of `object.mixin`](#5914).